apps/data-types.md

```diff
@@ -251,6 +259,31 @@ This object is used for the attributes of shippingAddress and billingAddress.
 | lineItemPrice | object of [price](page:apps-data-types#price) | The price of the line item. |
 | lineItemProductGuid | string | The identifier of the related product line item. A deposit is always bound to a product.|
 
+## facet
+
+| Attribute | Type | Description |
+| - | - |  - |
+| attributeName | string | The name of the filter. Can be *ListPrice*, *Manufacturer*, *CategoryID*, or any other attribute created by the merchant.  |
+| attributeType | string | The type of the filter. Is *Standard* for *Manufacturer*, *CategoryID*, or *Price*. Is *PreDefString* for any other attribute created by the merchant. |
```

Comment:
In the definition of attributeName you mention ListPrice and here only Price. Isn't it the same?

Reply:
👍